package.json

```diff
@@ -40,11 +40,13 @@
 		"@dfinity/candid": "^1.3.0",
 		"@dfinity/identity": "^1.3.0",
 		"@dfinity/ledger-icp": "next",
+		"@dfinity/ledger-icrc": "^2.4.0",
 		"@dfinity/principal": "^1.3.0",
 		"@dfinity/utils": "next",
 		"@esbuild-plugins/node-globals-polyfill": "^0.2.3",
 		"@esbuild-plugins/node-modules-polyfill": "^0.2.2",
 		"@rollup/plugin-inject": "^5.0.5",
+		"crc": "^4.3.2",
```

Comment:
Let's not use this package, use https://www.npmjs.com/package/@dfinity/ledger-icrc#gear-encodeicrcaccount instaed.

Reply:
Discussed irl